cmd/postcli/main.go

```diff
@@ -2,6 +2,7 @@ package main
 
 import (
 	"context"
+	"crypto/ed25519"
```

Comment:
Our `github.com/spacemeshos/ed25519` library has been archived because we are not using key extraction any more.

We were only using `ed25519.GenerateKey` which is the same code in both libraries, so this change is a `no-op` and gets rid of an external dependency.